socialhome/streams/app/components/StreamElement.vue

```diff
@@ -6,126 +6,66 @@
             v-bind="author"
             :is-user-author="isUserAuthor"
             :is-user-local="isUserLocal"
-            :current-browsing-profile-id="currentBrowsingProfileId"
-            :is-user-authenticated="isUserAuthenticated"
         >
         </author-bar>
-        <div class="grid-item-bar d-flex justify-content-start">
+        <reactions-bar
+            :id="id"
+            :replies-count="repliesCount"
+            :shares-count="sharesCount"
+            :has-shared="hasShared"
+        >
             <div class="mt-1">
-                <a :ref="contentUrl" :title="timestamp">
+                <a :href="contentUrl" :title="timestamp" variant="link" class="unstyled-link">
                     {{ humanizedTimestamp }}<span v-if="edited"> {{ editedText }}</span>
                 </a>
-                <div v-if="isUserAuthor">
-                    <a :href="updateUrl">
+                &nbsp;
+                <template v-if="isUserAuthor">
+                    <a :href="updateUrl" variant="link">
                         <i class="fa fa-pencil" title="Update" aria-label="Update"></i>
                     </a>
-                    <a :href="deleteUrl">
+                    &nbsp;
+                    <a :href="deleteUrl" variant="link">
```

Comment:
It's an `a` tag, does it need `variant`?

Reply:
Does it look like a typo? Hmm... :D